### DISCUSSION TITLE
Multiple select option picker

### DISCUSSION THREAD

**Discussion by Trifecta7** [Budibase/budibase#1863] [Ideas]
IS there a way in a form option picker for the user to be able to select multiple options?

Option Picker -> Fruit
  Apples
  Pears - Selected
  Strawberries
  Oranges - Selected

**Comment by mjashanks** [comment]
Not at the moment. And it's not as simple as it might seem.

We would need a few additional features to be able to support this as well
- An Array/List column type in Budibase tables e.g. `Text + allow multiple`
- Automations would need to understand an a List
- How to apply datasource filters with multiple values

So, there are a few things to think about.

It's a great suggestion though - I can see the utility. I'll mark this as an idea, so that we can see it in future roadmap discussions.
